### DISCUSSION TITLE
Godot 4 .NET6

### DISCUSSION THREAD

**Discussion by KeVs404** [godotengine/godot-proposals#5435] [General Discussions]
From my understanding, Godot 4 uses .NET 6 instead of mono now. Shouldn't the new Godot 4 .NET executables be called "Godot_v4.0-beta1_dotnet_win64.zip" instead of "Godot_v4.0-beta1_mono_win64.zip" ?
Not a game breaking issues but I just thought of it while downloading the beta. Correct me if I'm wrong.

**Comment by Calinou** [comment]
The internal name is still `mono` for now, but it'll be renamed to `dotnet` in the future.

**Reply by KeVs404** [reply]
Okay, I see. Thanks for replying!

**Comment by aXu-AP** [comment]
This problem shouldn't persist for too long. From beta release [blog post](https://godotengine.org/article/dev-snapshot-godot-4-0-beta-1):

> Currently, the .NET version of Godot still requires a separate build of Godot, just as in Godot 3. However, we are planning on unifying the editor so there won't be a standard and a .NET build anymore but a single editor that will download the necessary additional components when .NET is used.

**Reply by KeVs404** [reply]
Oh, that's just perfect!
